### PR TITLE
Describe RFC process

### PR DESCRIPTION
**Pull request by oudod**

### CODE REVIEW

**Review by alexauroradev — COMMENTED** [review]
LGTM after adding necessary info in TODOs:

First TODO with examples: you can use the ones that we've discussed today.

- **Services isolation**. A major improvement which is aimed at separating storage and execution environments of services.
- **Transaction finality**. Adding additional mechanisms for transaction finalization without inclusion to a blockchain.
- **Code parallelization**. Make Exonum able to run on several cores simultaneously.

Second todo: probably need something similar for "motion for final comment period" as in Rust and also info about when the number is attached.

**Review by alexauroradev — APPROVED** [review]
LGTM

File: README.md
Comment (slowli) [inline README.md]: I couldn't find out the license for the Rust RFCs. Are you sure we can use it?

File: README.md
Comment (slowli) [inline README.md]: Inconsistent indentation with other lists. Also: what does "API changes" entail? Changes in the public interface of the `exonum` crate?

File: README.md
Comment (slowli) [inline README.md]: Nitpick: s/makes/make/

File: README.md
Comment (slowli) [inline README.md]: Nitpick: unnecessary spacing.

File: README.md
Comment (slowli) [inline README.md]: Are you sure this is a good choice? Compare with [BIP licensing](https://github.com/bitcoin/bips/blob/master/bip-0002.mediawiki#bip-licensing).

**Review by slowli — COMMENTED** [review]
Mostly good, but I think RFC licensing needs to be reworked - it may be too restrictive in certain scenarios.

File: README.md
Comment (oudod) [inline README.md]: According to this comment https://github.com/rust-lang/rfcs/issues/1259#issuecomment-220753590, we are not first.

More info on `rust-lang/rfcs` licensing:
https://github.com/rust-lang/rfcs/blob/master/text/2044-license-rfcs.md
https://github.com/rust-lang/rfcs/pull/2075

File: README.md
Comment (Maria-Nosyk) [inline README.md]: > don't assign an RFC number yet

eliminate "an"; better "just yet"

File: README.md
Comment (Maria-Nosyk) [inline README.md]: > whether a Exonum developer has been assigned

a --> an

File: README.md
Comment (Maria-Nosyk) [inline README.md]: > the author of the RFC also write the implementation

write --> writes

File: README.md
Comment (Maria-Nosyk) [inline README.md]: > an RFC through to completion

an --> the

File: README.md
Comment (Maria-Nosyk) [inline README.md]: > should be new RFCs

better expand like "should be presented in the form of new RFCs"

File: README.md
Comment (Maria-Nosyk) [inline README.md]: > Maintainers of related repositories

The maintainers of the related repositories

File: README.md
Comment (Maria-Nosyk) [inline README.md]: > When a decision is made,

the decision

File: README.md
Comment (Maria-Nosyk) [inline README.md]: in thread --> in the thread
RFC repository --> the RFC repository

File: README.md
Comment (Maria-Nosyk) [inline README.md]: > thus that associated issue -->

the associated issue

File: README.md
Comment (Maria-Nosyk) [inline README.md]: an RFC --> the RFC

File: README.md
Comment (Maria-Nosyk) [inline README.md]: IMO "Help! This...." looks better

File: README.md
Comment (Maria-Nosyk) [inline README.md]: IMO "Help! This...." looks better

File: README.md
Comment (Maria-Nosyk) [inline README.md]: otherwise --> "in other words"

File: README.md
Comment (Maria-Nosyk) [inline README.md]: > the larger community -->

a larger...

**Pull request merged**
